src/style.css

```
@@ -0,0 +1,73 @@
+body {
+  margin: 0;
+}
+
+.snake-container {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+
+.snake-container > div {
```

```suggestion
.block {
```
Створи клас замість того, щоб використовувати селектори по тегам